canonical-web-and-design/jaas.ai
# Remove the user details pages

## Done
Replace the user details route with a 404
Removed the user links and details templates

## QA
- Go to /search?tags=monitoring and check usernames are not links
- Go to /u/filebeat-charmers and check that 404s

## Details
Fixes https://github.com/canonical-web-and-design/jaas.ai/issues/671
